Tests/SentryTests/SentryCrash/SentryCrashTests.m

```diff
@@ -0,0 +1,86 @@
+#import "FileBasedTestCase.h"
+#import "SentryCrash.h"
+#include "SentryCrashReportStore.h"
+
+@interface SentryCrashTests : FileBasedTestCase
+
+@end
+
+@interface SentryCrash (private)
+- (NSArray *)getScreenShots:(int64_t)reportID;
+@end
+
+@implementation SentryCrashTests
+
+- (void)tearDown
+{
+    [super tearDown];
+    NSFileManager *fileManager = [NSFileManager defaultManager];
+    [fileManager removeItemAtPath:self.tempPath error:nil];
+}
+
+- (void)test_getScreenshots_CheckName
+{
+    [self initReport:12 withScreenshots:1];
+
+    SentryCrash *sentryCrash = [[SentryCrash alloc]
+        initWithBasePath:[self.tempPath stringByAppendingPathComponent:@"Reports"]];
+    NSArray *files = [sentryCrash getScreenShots:12];
+
+    XCTAssertEqualObjects(files.firstObject,
+        [self.tempPath stringByAppendingPathComponent:
+                           @"Reports/AppName-report-000000000000000c-screenshots/0.png"]);
+
+    XCTAssertEqual(files.count, 1);
```

Comment:
`l`: I think it would be better to first check if there is 1 file otherwise `files.firstObject` already crashes. Same applies for other tests.
```suggestion
    XCTAssertEqual(files.count, 1);
    XCTAssertEqualObjects(files.firstObject,
        [self.tempPath stringByAppendingPathComponent:
                           @"Reports/AppName-report-000000000000000c-screenshots/0.png"]);
```